ban/core/models.py

```diff
@@ -117,6 +117,7 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
+    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
```

Comment:
In complement to "S", I see that "I" "L" and "B" are also present. I assume they removed confusing digit but preserved the matching letters?